pyhik/hikvision.py

```diff
@@ -186,6 +257,8 @@ def initialize(self):
         else:
             _LOGGING.debug('No Events available in dictionary.')
 
+        self.motion_detection = self.get_motion_detection()
```

Comment:
No need to assign the return to the variable since the function does it on it's own.  Just `self.get_motion_detection()` should be fine.